tests/Microsoft.DotNet.Framework.Docker.Tests/ImageTests.cs

```diff
@@ -331,7 +331,7 @@ private void VerifyHttpResponseFromContainer(string containerName, string urlPat
                 while (retries > 0)
                 {
                     retries--;
-                    Thread.Sleep(TimeSpan.FromSeconds(2));
+                    Thread.Sleep(TimeSpan.FromSeconds(5));
```

Comment:
This change can be reverted.  The original sleep count of 2 seconds works fine when the Dockerfile change is applied.